test/test-style.js

```diff
@@ -2,6 +2,10 @@
 const fs = require('fs');
 const path = require('path');
 const url = require('url');
+const { platform } = require('os');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
```

Comment:
Previously, `os` would be re‑imported for every single file.

This PR ensures that it’s only ever imported once, which should improve test speed somewhat.

Reply:
Cool, thank you!